src/components/Column/ColumnName/ColumnName.tsx

```diff
@@ -1,19 +1,52 @@
 import * as React from 'react';
 
 import './ColumnName.scss';
+import Input from '../../Input';
 
 export interface ColumnNameProps {
   title: string;
   count: number;
+  isAdmin: boolean;
+  onUpdateColumnName: (newName: string) => void;
 }
 
 const ColumnName: React.FunctionComponent<ColumnNameProps> = ({
   title,
-  count
+  count,
+  isAdmin,
+  onUpdateColumnName
 }) => {
+  const onBlur = (e: any) => {
+    if (e.target.value !== title) {
+      onUpdateColumnName(e.target.value);
+    }
+  };
+
   return (
-    <h1 className="column-name">
-      {title}
+    <h1 className="column-name__header">
+      {isAdmin && (
+        <Input
```

Comment:
@bitionaire I have removed the black border of the input, there is still a green animation on hover, I hope people will find it :) I think it would be kinda ugly if there would be an edit button near every column, more so if there are 4 columns... Could look bad in a screen share by admin